repositories.yml

```diff
@@ -3,49 +3,173 @@ section:
+        version: v1.0.0
+      - name: cyberark/cloudfoundry-conjur-buildpack
+        url: https://github.com/cyberark/cloudfoundry-conjur-buildpack
+        tool: Cloud Foundry
+        description: The Conjur buildpack brings the benefit of Summon to
+          your Cloud Foundry-deployed applications. Leverage your app's Conjur identity
+          to automatically inject the secrets your app needs into its environment
+          at runtime.
+        certification: "certified"
+        version: v2.1.1
+
+  - name: DevOps Tools
+    description: Conjur OSS integrations with DevOps tools.
+    repos:
+      - name: cyberark/ansible-conjur-host-identity
+        url: https://github.com/cyberark/ansible-conjur-host-identity
+        tool: Ansible
+        description: Ansible role to provide Conjur machine identity to application
+          hosts and install the Summon tool, which enables hosts to securely retrieve
+          credentials.
+        certification: "trusted"
+        version: v0.3.0
+      - name: cyberark/conjur-credentials-plugin
+        url: https://github.com/cyberark/conjur-credentials-plugin
+        tool: Jenkins
+        description: Conjur plugin for securely providing credentials to Jenkins jobs.
+        certification: "community"
+        version: v0.7
+      - name: cyberark/conjur-puppet
+        url: https://github.com/cyberark/conjur-puppet
+        tool: Puppet
+        description: Puppet module that simplifies the operation of establishing
+          Conjur host identity and allows authorized Puppet nodes to fetch secrets
+          from Conjur.
+        certification: "trusted"
+        version: v2.0.1
+      - name: cyberark/terraform-provider-conjur
+        url: https://github.com/cyberark/terraform-provider-conjur
+        tool: Terraform
+        description: Terraform provider that makes secrets in Conjur available in
+          Terraform manifests.
+        certification: "certified"
+        version: v0.1.0
+        
+  - name: Secretless Broker
+    description:
+    repos:
+      - name: cyberark/secretless-broker
+        url: https://github.com/cyberark/secretless-broker
+        description: Secretless Broker can be used to securely connect your
+          applications to services they need - without ever having to fetch or
+          manage passwords and keys.
+        certification: "certified"
+        version: v1.5.1
+          
+  - name: Summon
+    description:
+    repos:
+      - name: cyberark/summon
+        url: https://github.com/cyberark/summon
+        description: Summon is a secure tool to inject secrets into a subprocess
+          environment.
+        certification: "certified"
+        version: v0.8.0
+      - name: cyberark/summon-aws-secrets
+        url: https://github.com/cyberark/summon-aws-secrets
+        description: Summon provider for AWS Secrets Manager.
+        certification: "community"
+        version: v0.2.0
+      - name: cyberark/summon-s3
```

Comment:
Should this be at the end of the list (since it's alphabetically after the ones below it)?